pkg/server/events.go

```diff
@@ -366,12 +369,18 @@ func (b *backOff) reBackOff(key string, events []interface{}, oldDuration time.D
 }
 
 func (b *backOff) start() <-chan time.Time {
+	b.tickerMu.Lock()
+	defer b.tickerMu.Unlock()
 	b.ticker = time.NewTicker(b.checkDuration)
 	return b.ticker.C
 }
 
 func (b *backOff) stop() {
-	b.ticker.Stop()
+	b.tickerMu.Lock()
+	defer b.tickerMu.Unlock()
+	if b.ticker != nil {
+		b.ticker.Stop()
+	}
```

Comment:
else...logrus.WithError(err).Error("attempt to stop ticker before start has completed allocating the ticker")

note: should be impossible for this to happen now that you have a mutex... if the log is impossible we don't need the nil check..

Reply:
user interrupts during init of the plugins... interesting thx